targets/esp8266/readme.md

```diff
@@ -56,26 +36,36 @@ Default USB device is `/dev/ttyUSB0`. If you have different one, give with `USBD
 USBDEVICE=/dev/ttyUSB1 make -f ./targets/esp8266/Makefile.esp8266 flash
 ```
 
+### 4. Running
+
+* power off
+* connect GPIO2 with serial of 470 Ohm + LED and to GND
+* power On
+
+LED should blink on and off every second
+
+#### 5. Cleaning
 
-### 5. Running
+To clean the build result:
 
-1. Power off
-2. Disonnect(float) both GPIO0 and GPIO2
-3. Power on
+```
+make -f ./targets/esp8266/Makefile.esp8266 clean
+```
 
-Sample program here works with LED and a SW with below connection.
-* Connect GPIO2 to a LED > 4K resistor > GND
-* Connect GPIO0 between VCC > 4K resistor and GND
+To clean the board's flash memory:
+```
+make -f ./targets/esp8266/Makefile.esp8266 erase_flash
+```
 
-If GPIO0 is High then LED is turned on longer. If L vice versa.
 
 ### 6. Optimizing initial RAM usage (ESP8266 specific)
-The existing open source gcc compiler with Xtensa support stores const(ants) in
-the same limited RAM where our code needs to run. 
+The existing open source gcc compiler with Xtensa supports storing const(ants) in
```

Comment:
Oh, I've misread the sentence last time. It is OK as it was. "gcc compiler with Xtensa support stores const(ants) in the same limited RAM" is all right. Please, revert to that version. Sorry, my bad.